### PR TITLE
solved task

### PR DESCRIPTION
**Pull request by mykolazx3**

### CODE REVIEW

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 1]
```diff
@@ -11,6 +11,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
```
Comment (karma-o) [inline src/main/java/mate/academy/service/AuthenticationService.java:14]: let's not create a new service each time we call a `login()` method, and make it an instance level variable of `AuthenticationService`

**Review by karma-o — CHANGES_REQUESTED** [review]
GJ! Left a small comment to improve your solution

File: src/main/java/mate/academy/service/AuthenticationService.java (hunk 1 above)
Comment (mykolazx3) [reply src/main/java/mate/academy/service/AuthenticationService.java]: > let's not create a new service each time we call a `login()` method, and make it an instance level variable of `AuthenticationService`

fixed

**Comment by mykolazx3** [comment]
> GJ! Left a small comment to improve your solution

comment was added

**Comment by mykolazx3** [comment]
??

**Comment by karma-o** [comment]
> I saw you rejected my fixed task solution again, why??

It was not a mentor rejection, it was an automated CI check that failed because you made a mistake in the checkstyle, (see the screenshot)
image
Red cross there is clickable where you can see the details of why that happened

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 2]
```diff
@@ -10,7 +10,15 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
```
Comment (karma-o) [inline src/main/java/mate/academy/service/AuthenticationService.java:12]: This code section is a Javadoc of the `login()` method and should be right above it, class fields on the other hand should be at the beginning of the class

**Review by karma-o — APPROVED** [review]
GJ! Left a comment about the rejection that you have asked, and another minor thing

**Pull request closed without merging**